src/components/SlateEditor/plugins/h5p/EditH5P.tsx

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+import { ReactEditor } from 'slate-react';
+import { Editor, Path, Transforms } from 'slate';
+import { useTranslation } from 'react-i18next';
+import { ChangeEvent, useCallback, useState } from 'react';
+import { H5pEmbedData, H5pMetaData } from '@ndla/types-embed';
+import { ModalContent } from '@ndla/modal';
+import styled from '@emotion/styled';
+import { ButtonV2 } from '@ndla/button';
+import { TextAreaV2 } from '@ndla/forms';
+import { H5pElement } from './types';
+import H5PElement, { OnSelectObject } from '../../../H5PElement/H5PElement';
+import config from '../../../../config';
+import { getH5pLocale } from '../../../H5PElement/h5pApi';
+import { EditingState } from './SlateH5p';
+import FormikFieldDescription from '../../../FormikField/FormikFieldDescription';
+
+const StyledModalBody = styled.div`
+  display: flex;
+  height: 100%;
+  flex-direction: column;
+  padding: 50px;
+  gap: 30px;
```

Comment:
Hvor redde er vi for magic-numbers som disse? Bør/må det være en eller annen magisk spacing verdi fra et annet sted?

Reply:
Det må ikke. Jeg er egentlig litt skeptisk til at spacing brukes til alt, overalt. Men det er sikkert bedre å bruke de enn å velge verdier på måfå.